### DISCUSSION TITLE
Trying to use ncu with bunx but getting a weird error

### DISCUSSION THREAD

**Discussion by MordechaiHadad** [raineorshine/npm-check-updates#1426] [Q&A] [answered]
I am not sure if this is worthy of an issue, but if it is I will open one.

When I am trying to use ncu I get this:
```
PS D:\repos\LeadLens\gui> bunx ncu --doctor
NCU Real Time Weather is Running......
2024/6/16 14:4 Sunday UTC+8:00

(node:14304) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Whoops! Please check your network connection. OR may be http://pblap.atm.ncu.edu.tw is dead, error message:

AggregateError
```

What exactly is the cause of this?

**Comment by raineorshine** [comment, marked as answer]
When npm-check-updates is installed globally you can use the alias `ncu`, but with `npx` or `bunx` you must use the full name: `bunx npm-check-updates`. 

[ncu](https://www.npmjs.com/package/ncu) is a deprecated weather app for the National Central University, Taiwan.

**Reply by MordechaiHadad** [reply]
Oh my bad, thanks for clarifying
